tutorials/harfbuzz/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "icu_harfbuzz_tutorial"
+publish = false
+license = "Unicode-3.0"
+edition = "2021"
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+icu_harfbuzz = { version = "0.1.2", features = ["compiled_data"] }
```

Comment:
Nit: Double check how we handle local paths in the other tutorial crates. Note that this won't work as written until we publish `icu_harfbuzz`.

Reply:
Yeah we do a patch, it works here